tests/schema.js

```diff
@@ -72,7 +66,7 @@ test('Expected table config', (t) => {
  * @typedef {Extract<MapeoDoc, { schemaName: T }>} MapeoType
  */
 
-test('Types match', { skip: true }, (t) => {
+test('Types match', { skip: true }, () => {
```

Comment:
completely unrelated, but wondering if this belongs in `test-types/` 🤔

Reply:
Ask and ye shall receive: #657